README.md

```diff
@@ -171,42 +171,41 @@ ember exam --split=3 --partition=3
 $ ember exam --load-balance=<num>
 ```
 
-`load-balance` allows you to specify a number of browser(s) to load balance test files against. It will order the test files by test types with test load balancing. If you specify a value for load-balance of 3, then three browser instances will be created and the output looks something like:
+`load-balance` allows you to specify a number of browser(s) to load test files against. It will order the test files by test types with test load balancing. For example, you specify load-balance equals 3, then three browser instances will be created and the output will look something like:
 
 ```bash
 ok 1 Chrome 66.0 - Browser Id 1 - some test
 ok 2 Chrome 66.0 - Browser Id 2 - some another test
 ok 3 Chrome 66.0 - Browser Id 3 - some the other test
 ```
 
-You can also specify `split` and `partition` option with `load-balance` to load partion of AMD modules balanced.
+You can also specify the `split` and `partition` options with `load-balance` to load a portion of AMD modules to each machines.
 
 ```bash
 $ ember exam --split=<num> --partition=<num> --load-balance=<num>
 ```
 
-This command will split AMD modules and load specified test groups balanced. If you specify a split of 2, a partition of 1, and a value for load-balance of 3 the AMD modules are split into 2 and the first portion of the modules is loaded with three browsers. The output looks something like below:
+This command will split AMD modules and load-balance tests from specified partition across the browsers. If you specify a split of 2, a partition of 1, and a value for load-balance of 3 the AMD modules are split into 2 and the first half is loaded into three browsers. The output will look something like below:
 ```base
 ok 1 Chrome 66.0 - Exam Partition 1 - browser Id 1 - some test
 ok 2 Chrome 66.0 - Exam Partition 1 - browser Id 2 - some another test
 ok 3 Chrome 66.0 - Exam Partition 1 - browser Id 3 - some the other test
 ```
 
-If `load-balance` is used without specifying a number of browser(s), the option must be passed the last. Otherwise, any following arguments will be read as a value for `--load-balance`. For instances,
+If `load-balance` is used without specifying a number of browser(s), the option must be passed last and the option has default value of 1. Otherwise, any following arguments will be read as a value for `--load-balance`. For instance:
```

Comment:
I would change the order a bit:
  the option has a default value of 1 and must be passed last.